README.md

```diff
@@ -112,69 +253,92 @@ synopsys_scan synopsys_security_product: "COVERITY"
+| `synopsys_bridge_download_version`  | Provide bridge version. If provided, the specified version of Synopsys Bridge will be downloaded and configured.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| `include_diagnostics`               | If this is set **true** then the detailed bridge logs will be shown in console and bridge diagnostics will be uploaded in Jenkins Archive Artifact.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| `network_airgap`                    | When **network_airgap** is set **true**, <br/> 1. If Bridge exists in the default `home/USER/synopsys-bridge` or Bridge exists in the specified `<synopsys_bridge_install_directory>`, it is used. Otherwise, errors out. <br/> 2. When `<synopsys_bridge_download_url>` provided by user, <br/> a. If Bridge version specified by `<synopsys_bridge_download_url>` is the same version that already exists in default `<synopsys_bridge_install_directory>`, it is used.<br/> b. If not, Bridge version pointed to by `<synopsys_bridge_download_url>` is downloaded to the default `<synopsys_bridge_install_directory>` and used/cached.<br/> c. If `<synopsys_bridge_download_url>` doesn't have version info, plugin will look for versions.txt file at the same download URL folder level. If versions.txt file is not found, Bridge is not cached. |           
```

Comment:
@kishorikumar, any more suggestions?

Reply:
Bridge version specified by can be rephrased as "Bridge version available in Synopsys Bridge url". 

default <synopsys_bridge_install_directory> - <synopsys_bridge_install_directory> is not a default path, instead <synopsys_bridge_install_directory> is used to define custom install directory.